auto-remediate/lambda_handler.py

```diff
@@ -61,6 +61,8 @@ def remediate(self):
                             remediation = self.security_hub.s3_bucket_public_read_prohibited(config_rule_resource_id)
                         elif 's3-bucket-public-write-prohibited' in config_rule_name:
                             remediation = self.security_hub.s3_bucket_public_write_prohibited(config_rule_resource_id)
+                        elif 's3-bucket-logging-enabled' in config_rule_name:
```

Comment:
I think this section is getting too messy. Can we try to refactor the core of this function (routing of the remediation based on rule name) like this

```python3
Class Remediate:
    def __init__ ...

        self.remediations = {
            "securityhub-restricted-rdp": self.security_hub.restricted_rdp,
            ...
        }

    def remediate(remediation_rule):
        return self.remediations.get(remediation_rule, rem_not_found_function)()

```

Reply:
I've created a new issue (#21) for this refactoring, I'll implement it soon.